UnleashClient/strategies/Strategy.py

```diff
@@ -24,9 +33,13 @@ def __init__(
         parameters: dict = {},
         segment_ids: list = None,
         global_segments: dict = None,
+        variants: list = None,
     ) -> None:
+        if variants is None:
+            variants = []
         self.parameters = parameters
         self.constraints = constraints
+        self.variants = variants
```

Comment:
Just to remain stylistically consistent with the other arguments here, how about this?

```suggestion
        self.parameters = parameters
        self.constraints = constraints
        self.variants = variants or []
```

Reply:
Good suggestion, has the nice side effect of defaulting this to an empty list if someone passes hot garbage like `False` here and isn't using a static type checker

Reply:
Nice, sure